### DISCUSSION TITLE
Unhandled Runtime Error TypeError: Cannot read properties of undefined (reading 'connector')

### DISCUSSION THREAD

**Discussion by saidul112** [WalletConnect/walletconnect-monorepo#3946] [Developer Support]
Hey, I am using https://docs.walletconnect.com/web3modal/react/about?platform=wagmi into my React project. I have integrated in one file. But during run time i get this error "Unhandled Runtime Error TypeError: Cannot read properties of undefined (reading 'connector')"

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/87698836/0a4c2992-2163-4a9f-a830-65b25842036f)
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/87698836/60ed4bb7-0258-4b5f-ab1b-c1f30c697876)

**Comment by crypblizz8** [comment]
Are you potentially using Next 14?

**Reply by pro123908** [reply]
Getting same error on react 18

**Comment by jfschwarz** [comment]
I'm getting the same error. Using Next 12

**Reply by jfschwarz** [reply]
Any hints for resolving it?
